sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClientTest.java

```diff
@@ -722,6 +724,39 @@ public void verifyRequestFailures(
         }
     }
 
+    // TODO: add validations for other properties
+    @Test(groups = "unit")
```

Comment:
I think a test case is missing where connect timeout is < default connect timeout

Reply:
added the missing test
But I disabled the "transportClientCustomizedOptionsTests" for now as had a hard time for forcing the class to be reloaded - @FabianMeiswinkel  any suggestions here?